src/main/java/com/staffmanagement/dto/UserDTO.java

```diff
@@ -0,0 +1,45 @@
+package com.staffmanagement.dto;
+
+public class UserDTO {
+
+	private String name;
+
+	private String profession;
+
+	private String age;
+
+	private String address;
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public String getProfession() {
+		return profession;
+	}
+
+	public void setProfession(String profession) {
+		this.profession = profession;
+	}
+
+	public String getAge() {
+		return age;
+	}
+
+	public void setAge(String age) {
+		this.age = age;
+	}
+
+	public String getAddress() {
+		return address;
+	}
+
+	public void setAddress(String address) {
+		this.address = address;
+	}
+
```

Comment:
it is also recommended to always override hashcode, equals and tostring. may be u can use lombok annotations for this purpose.